src/oedisi/types/data_types.py

```diff
@@ -116,7 +116,7 @@ class EquipmentArray(MeasurementArray):
         "ImpedanceMagnitude",
         "ImpedanceAngle",
         "ImpedanceReal",
-        "ImpedanceImaginary",
+        "ImpedanceImaginary",.
```

Comment:
This looks like a weird ruff quirk. Maybe we should remove the last commas?

Reply:
Yeah I removed it. It's because the first part of a doc-string should end in a period.